sysl2/util.go

```diff
@@ -1,11 +1,13 @@
 package main
 
 // Set is Not exported
-type Set map[int]bool
+type Set map[int]struct{}
+
+var empty struct{}
```

Comment:
Just use the literal `struct{}{}` instead of a variable. Instances of `struct{}` are free.

(Side note: `const` would be better than `var` here.)

Reply:
How do you declare a const struct{} ?

Reply:
used struct{}{}